stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -226,7 +238,7 @@ end
 Base.show(io::IO, S::AbstractSparseMatrixCSC) = Base.show(convert(IOContext, io), S::AbstractSparseMatrixCSC)
 
 const brailleBlocks = UInt16['⠁', '⠂', '⠄', '⡀', '⠈', '⠐', '⠠', '⢀']
-function _show_with_braille_patterns(io::IOContext, S::AbstractSparseMatrixCSC)
+function _show_with_braille_patterns(io::IOContext, S::AbstractSparseMatrixCSCInclAdjointAndTranspose)
```

Comment:
This linear algebra aliases are getting kind of ridiculous. Not a critique, just a humorous observation.

Reply:
Yes, I'm aware of that, I also don't like it.
If anybody has a better solution, feel free to suggest something.

Reply:
Not to be tackled in this PR!